pkg/apis/networking/v1beta1/conversion.go

```diff
@@ -75,3 +76,18 @@ func Convert_networking_IngressSpec_To_v1beta1_IngressSpec(in *networking.Ingres
 	}
 	return nil
 }
+
+func Convert_v1_TypedLocalObjectReference_To_networking_IngressClassParametersReference(in *core.TypedLocalObjectReference, out *networking.IngressClassParametersReference, s conversion.Scope) error {
+	if in.APIGroup != nil {
+		out.APIGroup = &(*in.APIGroup)
+	}
+	out.Kind = in.Kind
+	out.Name = in.Name
+	return nil
+}
+
+func Convert_networking_IngressClassParametersReference_To_v1_TypedLocalObjectReference(out *networking.IngressClassParametersReference, in *core.TypedLocalObjectReference, s conversion.Scope) error {
+	// TODO(hbagdi): figure out what to do when namespace is set?
+	// I don't understand why we need these functions in the first place
+	return nil
```

Comment:
I'm not sure what should I be doing here. Can anyone provide some guidance?

Reply:
Although you could write custom conversion logic here, I think it may be better to just use the new reference type for v1beta1 as well since it's entirely additive.